docs/src/configuring.adoc

```diff
@@ -0,0 +1,292 @@
+# Configuring Cassandra Sidecar
```

Comment:
As I was writing this, I was questioning the value of listing out all YAML options the way we do for Cassandra (https://cassandra.apache.org/doc/5.0/cassandra/managing/configuration/cass_yaml_file.html) or if the source of truth should just be the YAML with comments explaining how to use each value. I could see this going out of date pretty quickly.